userspace/libsinsp/sinsp_filter_transformer.cpp

```diff
@@ -180,6 +193,20 @@ bool sinsp_filter_transformer::transform_values(std::vector<extract_value_t>& ve
         }
         return true;
     }
+    case FTR_BASENAME:
+    {
+        return string_transformer(vec, t, [](std::string_view in, storage_t& out) -> bool {
+            auto last_slash_pos = in.find_last_of("/");
```

Comment:
Question, why can't we directly use posix `basename` here?

Reply:
Because we also build for Windows and MacOS, which are not guaranteed to have POSIX compatible functions. Especially on Windows a basename function would expect backslashes instead of slashes potentially making reading Linux scap files a bit of a headache.